README.md

```diff
@@ -12,12 +12,12 @@ limited. We plan to add more content and features as we develop towards version
 
 ## User setup
 
-1. From https://github.com/google/tie/, click on the green "Clone or download"
-button then click "Download ZIP"
+1. From [https://github.com/google/tie/](https://github.com/google/tie/), click on the green ``Clone or download``
```

Comment:
Just a note -- for this and other "code font" stuff, isn't just one backtick enough? I.e. "`" rather than "``" (at both the start and end)?